### PR TITLE
Docs: Add instructions for installing plugins/themes with wp-env

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Description

This PR adds an example for installing plugins and themes using `wp-env`. I encountered an issue similar to wp-cli/wp-cli#3255, docker-library/wordpress#417 where I was unable to install plugins or themes. I've provided instructions for how to install plugins using the conventional `wp-env` method, as well as an alternate in case that does not work for some folks.

## How has this been tested?

The additional command I provided was the only way I was able to install a plugin locally; I've tested that both themes and plugins can be installed with this method.

## Types of changes

Small docs addition!

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal). 

(I've left these unchecked since they don't appear to apply to documentation, but please let me know if there are any I should check/run)

### CODE REVIEW

File: packages/env/README.md [hunk 1]
```diff
@@ -350,6 +350,29 @@ wp> ^C
...
+Downloading installation package from https://downloads.wordpress.org/plugin/custom-post-type-ui.zip...
+The authenticity of custom-post-type-ui.zip could not be verified as no signature was found.
+Unpacking the package...
+Installing the plugin...
+Plugin installed successfully.
+Success: Installed 1 of 1 plugins.
+✔ Ran `plugin install custom-post-type-ui` in 'cli'. (in 6s 483ms)
+```
+
+**NOTE**: Depending on your host OS, you may experience errors when trying to install plugins or themes (e.g. `Warning: Could not create directory.). This is typically because the user ID used within the container does not have write access to the mounted directories created by `wp-env`. To resolve this, run the `docker-compose` command directly from the directory created by `wp-env` and add `-u $(id -u)` and `-e HOME=/tmp` the `run` command as options:
```
Comment (Mamaduka) [inline packages/env/README.md]: I think backtick is missing after "`Warning: Could not create directory." and results in misformatted text in note section - [rendered version.](https://github.com/WordPress/gutenberg/blob/800f97028f2329763885f50717218e42fb8d9c5d/packages/env/README.md#installing-a-plugin-or-theme-on-the-development-instance)

Comment (AetherUnbound) [reply packages/env/README.md]: Gah, thanks for catching that! I've updated the page to include that backtick.

**Review by Mamaduka — APPROVED** [review]
Thanks for working on this, @AetherUnbound.

Let's merge this PR once all checks are green ✅

**Comment by AetherUnbound** [comment]
Fantastic, thanks for the quick turnaround! 🙌🏼

**Comment by oxyc** [comment]
Trying to run `wp-env` on Github Actions and running into this but the workaround doesn't fix it either.

**Pull request merged**